src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -63,6 +63,8 @@ service Node {
 	void repair() throws (1: Error err)
 	TruncateResult truncate(1: TruncateRequest req) throws (1: Error err)
 
+    AggregateTilesResult aggregateTiles(1: AggregateTilesRequest req) throws (1: Error err)
+
```

Comment:
Here too: Do you want to format everything to use tabs instead of spaces in this file? Looks like we are mixing them.

Reply:
👍